### PR TITLE
LPS-169752 When the title field is a system field the asset title is null on the My Workflow Tasks

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
https://issues.liferay.com/browse/LPS-169752

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectEntryImpl.java [hunk 1]
```diff
@@ -84,8 +86,20 @@ public String getTitleValue() throws PortalException {
 					objectDefinition.getTitleObjectFieldId());
 
 			if (objectField != null) {
+				Map<String, Serializable> values = Stream.of(
```
Comment (p-albuquerque) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectEntryImpl.java:89]: Unfortunately, Brian don't let pass code using **stream** anymore.
You could try use something like:
```
HashMapBuilder.create(
			getValues()
		).putAll(
			ObjectEntryLocalServiceUtil.getSystemValues(this)
		).build()
```

**Review by p-albuquerque — CHANGES_REQUESTED** [review]

**Review by p-albuquerque — CHANGES_REQUESTED** [review]

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Review by p-albuquerque — APPROVED** [review]

**Pull request closed without merging**